csrc/python_frontend/fusion_record.h

```diff
@@ -727,47 +727,36 @@ struct SqueezeOpRecord : RecordFunctor {
 
 //! Specialized Record Functor for the FusionState's broadcast_in_dim op.
 
-template <typename OutputShapeType>
 struct BroadcastInDimOpRecord : RecordFunctor {
```

Comment:
Removing the template argument for the `BroadcastInDim` Fusion Record because the record only consumes a Vector (found in State) instead of a distinct attribute for either a `List[int]` or a `List[Scalar]`.